File: platform/android/MapLibreAndroid/src/vulkan/java/org/maplibre/android/maps/renderer/surfaceview/VulkanSurfaceViewMapRenderer.java
package org.maplibre.android.maps.renderer.surfaceview;

import android.content.Context;
import androidx.annotation.NonNull;

public class VulkanSurfaceViewMapRenderer extends SurfaceViewMapRenderer {

  public VulkanSurfaceViewMapRenderer(Context context,
                                @NonNull MapLibreVulkanSurfaceView surfaceView,
                                String localIdeographFontFamily) {
    super(context, surfaceView, localIdeographFontFamily);

    this.surfaceView.setRenderer(this);
  }

}

File: platform/android/MapLibreAndroid/src/sharedRenderer/vulkan/java/org/maplibre/android/maps/renderer/surfaceview/VulkanSurfaceViewMapRenderer.kt
package org.maplibre.android.maps.renderer.surfaceview

import android.content.Context

open class VulkanSurfaceViewMapRenderer(
    context: Context,
    surfaceView: MapLibreVulkanSurfaceView,
    localIdeographFontFamily: String?,
) : SurfaceViewMapRenderer(context, surfaceView, localIdeographFontFamily) {
    init {
        this.surfaceView.setRenderer(this)
    }
}
